src/MusicStore/test/MusicStore.Test/CartSummaryComponentTest.cs

```diff
@@ -1,90 +0,0 @@
-using System;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc.Rendering;
-using Microsoft.AspNetCore.Mvc.ViewComponents;
-using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.DependencyInjection;
-using MusicStore.Controllers;
-using MusicStore.Models;
-using Xunit;
-
-namespace MusicStore.Components
-{
-    public class CartSummaryComponentTest
-    {
-        private readonly IServiceProvider _serviceProvider;
-
-        public CartSummaryComponentTest()
-        {
-            var efServiceProvider = new ServiceCollection().AddEntityFrameworkInMemoryDatabase().BuildServiceProvider();
-
-            var services = new ServiceCollection();
-
-            services
-                .AddMemoryCache()
-                .AddLogging()
-                .AddDbContext<MusicStoreContext>(b => b.UseInMemoryDatabase("Scratch").UseInternalServiceProvider(efServiceProvider));
-
-            _serviceProvider = services.BuildServiceProvider();
-        }
-
-        [Fact]
-        public async Task CartSummaryComponent_Returns_CartedItems()
```

Comment:
Just so I know, why were these test dropped?

Reply:
I didn't spend an inordinate time on it, but it looked like incremental builds for the solution were broken because the unit tests would get x86 versions of the binaries and would then fail in exotic ways. We could spend time fixing this, but the tests really offer no value - they're unit testing a sample that we use for product testing.